### PR TITLE
Fix/take screenshot puppeteer

### PR DESCRIPTION
**Pull request by tourdedave**
[Trello](https://trello.com/c/SiYAh0Bh)

### CODE REVIEW

File: pending-changes.yaml [hunk 1]
```diff
@@ -61,6 +61,7 @@ Apollo SDK:
   bug-fix:
     - Fixed bug where a failure in a single UFG environment fails all other environments in the same configuration
     - Better support in DOM slot element
+    - Fixed issue that prevented screenshots from being taken
```
Comment (tourdedave) [inline pending-changes.yaml:64]: @benbabayoff8 ☝️

Comment (benbabayoff8) [reply pending-changes.yaml]: @tourdedave Looks good

**Comment by tourdedave** [comment]
I tested this fix against the coverage tests for eyes-puppeteer and they passed

https://github.com/applitools/eyes.sdk.javascript1/runs/7449133781?check_suite_focus=true

**Comment by kyryloonufriiev** [comment]
@tourdedave I find a way to create a test that will work for both browsers. I think it could be merged

**Pull request merged**